interfaces.ts

```diff
@@ -0,0 +1,14 @@
+enum Type {
+    host,
+    participant
+}
+
+export interface user {
+    id: string;
+    firstname?: string;
+    lastname?: string;
+    AKA?: string;
+    dateOfBirth?: Date;
+    type: Type
+}
+
```

Comment:
1. move it to proper folder, not the root level
2. `Type` should be something like `UserType` + please use something like `type UserType = "host" | "participant"` instead of enum
3. interface or type should be Uppercase like `User`